lib/components/mobile/navigation-bar.js

```diff
@@ -110,6 +112,7 @@ class MobileNavigationBar extends Component {
 const mapStateToProps = (state) => {
   return {
     auth0Config: getAuth0Config(state.otp.config.persistence),
+    config: state.otp.config,
```

Comment:
Once accountLinks is extracted this can be refactored to only extract the part of the config we need